winpr/libwinpr/clipboard/synthetic_file.c

```diff
@@ -330,7 +328,6 @@ static BOOL do_add_directory_contents_to_list(wClipboard* clipboard, const WCHAR
 		BOOL bRet = FindNextFileW(hFind, &FileData);
 		if (!bRet)
 		{
-			WLog_WARN(TAG, "FindNextFile failed (%" PRIu32 ")", GetLastError());
```

Comment:
Maybe, it would be sufficient to just move this warning a few lines below after the `if (ERROR_NO_MORE_FILES == GetLastError())` check.